db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -8,9 +8,13 @@ def request_info
   $evm.log(:info, "Request: #{@miq_request.description} id: #{@miq_request.id} ")
 end
 
+def cloud?(prov_type)
+  %w(amazon openstack).include?(prov_type)
```

Comment:
I would like to address so we are not dependent and a static list (we will be added Azure soon), but that can be a separate PR.